src/features/people/PeopleDetails/index.js

```diff
@@ -24,12 +24,12 @@ const PersonDetails = () => {
             : status === "error" ? <div>Error!</div>
                 : details ? (
                     <>
-                        <PersonDetailsTile
-                            poster={details.profile_path}
+                        <DetailsTile
+                            posterPath={details.profile_path}
                             title={details.name}
-                            overview={details.biography}
-                            dateOfBirth={details.birthday}
-                            placeOfBirth={details.place_of_birth}
+                            description={details.biography}
+                            firstData={details.birthday}
```

Comment:
remember to format the date, I suggest running API response through a function to modify the raw data

Reply:
Thanks for the heads up! I noticed your suggestion about date formatting and agree that it's an important improvement. I will address this in future improvements and apply the solution with a function analogous to your solution.
Your input is valuable and I'm committed to work out the details to get a refined result 😊🚀